eng/pipelines/templates/common.yml

```diff
@@ -170,7 +170,16 @@ extends:
               VsixTargets: ${{ parameters.VsixTargets }}
               ServerName: ${{ parameters.ServerName }}
               IncludeNative: ${{ parameters.IncludeNative }}
+          - powershell: |
```

Comment:
This should really be the responsibility of docker.yml.  We should let package specific logic bleed into common.yml